custom/icds_reports/views.py

```diff
@@ -1599,6 +1600,7 @@ class ICDSAppTranslations(BaseDomainView):
     page_title = ugettext_lazy('ICDS App Translations')
     urlname = 'icds_app_translations'
     template_name = 'icds_reports/icds_app/app_translations.html'
+    section_name = ugettext_noop("Translations")
```

Comment:
Shouldn't this be `ugettext_lazy` like `page_title`?

Reply:
Should this always be `ugettext_lazy`. we have used `ugettext_noop` at places for section names as well.

Reply:
Oh, actually, you're right - looks like `ugettext_noop` should work:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/standard/base_template.html#L69-L69
It should only be used in variables which are explicitly translated later.

I'm not convinced we need to use `ugettext_noop` in this instance, and doing so may mean that it doesn't get translated when the report is rendered through other means, like when it's exported or emailed, unless it's also translated there (I didn't see any other relevant matches for `trans.*section_name`, `_\(.*section_name`, or `ugettext\(.*section_name`).  That however, is a battle for another day, as this is clearly meant to accept `ugettext_noop`.